src/App.tsx

```diff
@@ -15,36 +15,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  None = '',
+  Alphabetically = 'alphabetically',
+  ByLength = 'length',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: string,
```

Comment:
The `sortField` parameter should use the `SortType` enum instead of a string. This ensures type safety and consistency across the code.